runtime/Cpp/runtime/src/misc/MurmurHash.cpp

```diff
@@ -17,118 +20,83 @@ using namespace antlr4::misc;
 
 #if defined(_MSC_VER)
 
-#define FORCE_INLINE	__forceinline
-
 #include <stdlib.h>
 
 #define ROTL32(x,y)	_rotl(x,y)
 #define ROTL64(x,y)	_rotl64(x,y)
 
-#define BIG_CONSTANT(x) (x)
+#elif ANTLR4_HAVE_BUILTIN(__builtin_rotateleft32) && ANTLR4_HAVE_BUILTIN(__builtin_rotateleft64)
+
+#define ROTL32(x, y) __builtin_rotateleft32(x, y)
+#define ROTL64(x, y) __builtin_rotateleft64(x, y)
 
 #else	// defined(_MSC_VER)
 
 // Other compilers
 
-#define	FORCE_INLINE inline __attribute__((always_inline))
+namespace {
```

Comment:
Why that namespace? Nothing changed which would cause a violation of the ODR.

Reply:
We are defining it in the global namespace, so putting it in an anonymous namespace or declaring it static is best practice, makes it very obvious that this is only relevant for this translation unit. These days anonymous namespace are preferred over `static`.